src/main/resources/application.properties

```diff
@@ -57,12 +57,12 @@ fusionauth.key = ${FUSIONAUTH_KEY}
 # XMessage user data before hours from local current time - float value allowed 
 xmessage.user.data.before.hours=20.5
 
-spring.r2dbc.url=r2dbc:postgresql://${FORMS_DB_HOST}:${FORMS_DB_PORT}/${FORMS_DB_NAME}
-postgresql.db.host=${FORMS_DB_HOST}
-postgresql.db.port=${FORMS_DB_PORT}
-spring.r2dbc.name=${FORMS_DB_NAME}
-spring.r2dbc.username=${FORMS_DB_USERNAME}
-spring.r2dbc.password=${FORMS_DB_PASSWORD}
+#spring.r2dbc.url=r2dbc:postgresql://${FORMS_DB_HOST}:${FORMS_DB_PORT}/${FORMS_DB_NAME}
+#postgresql.db.host=${FORMS_DB_HOST}
+#postgresql.db.port=${FORMS_DB_PORT}
+#spring.r2dbc.name=${FORMS_DB_NAME}
+#spring.r2dbc.username=${FORMS_DB_USERNAME}
+#spring.r2dbc.password=${FORMS_DB_PASSWORD}
```

Comment:
What do these settings correspond to and why have we removed them?

Reply:
These configurations are related to the transformer service and we are not using the transformer in inbound. Also, these configurations are added in the other services like Broadcast-Transformer, Orchestrator, and Outbound which we need to remove because formsDB is used only in the transformer.

Reply:
Got it, then please remove them altogether instead of commenting them out.

Reply:
Removed commented configuration from inbound